doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -25,59 +26,61 @@ This should make it easy for you to develop changes to this site and make sure y
 
 1. Build and view the website locally
 
-To build the website locally run this command:
+  To build the website locally run this command:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  make html
-  # Or run the following line if you want to automatically rebuild the website on new changes
-  while inotifywait -re modify,move,create,delete .; do
     make html
-  done
+    # Or run the following line if you want to automatically rebuild the website on new changes
+    while inotifywait -re modify,move,create,delete .; do
+      make html
+    done
 
-Then you can open the local build of the site in your web browser from: ``./build/html/index.html``. For example, to open the local site in Firefox the command would be:
+  Then you can open the local build of the site in your web browser from: ``./build/html/index.html``. For example, to open the local site in Firefox the command would be:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  firefox ./build/html/index.html
+    firefox ./build/html/index.html
 
 2. Test for broken links
 
-To test for broken links run this command:
+  To test for broken links run this command:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  ./htmlprofer.sh
+    ./htmlproofer.sh
 
 3. Run the formatters and spellchecker
 
-We use `pre-commit <https://pre-commit.com/>`_ to run the formatting and spelling checkers.
-You can install it with pip like this:
+  We use `pre-commit <https://pre-commit.com/>`_ to run the formatting and spelling checkers.
+  You can install it with pip like this:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  python3 -m pip install --user pre-commit
+    python3 -m pip install --user pre-commit
 
-To run pre-commit locally to fix formatting and spelling:
+  To run pre-commit locally to fix formatting and spelling:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  pre-commit run --all
+    pre-commit run --all
 
 4. Run industrial_ci locally to run CI
 
-1. Clone a copy of `industrial_ci <https://github.com/ros-industrial/industrial_ci>`_ into your workspace.
-2. Build and source your workspace.
-3. Run this command from the workspace directory to test code changes just as it is done in CI:
+  - Clone a copy of `industrial_ci <https://github.com/ros-industrial/industrial_ci>`_ into your workspace.
```

Comment:
This is the first time you mention workspace. So far, you could follow along without having to create a ros2 workspace. So add a step to the very beginning (before cloning this repo) that tells the user to first create a ros2 workspace.

Reply:
I re-phrased the Requirements section to clarify this.